### ISSUE TITLE
Uncaught TypeError: Cannot access offset of type string on string (PHP81)

### ISSUE THREAD

**Issue by tertek** [susom/mass-delete#10]
Experiencing Uncaught TypeError on PHP81 when clicking delete.

```
PHP Fatal error: Uncaught TypeError: Cannot access offset of type string on string in /var/www/html/redcap/modules/mass_delete_v1.1.3/MassDelete.php:229 
Stack trace: #0 /var/www/html/redcap/modules/mass_delete_v1.1.3/MassDelete.php(54): Stanford\MassDelete\MassDelete->handleDelete()
 #1 /var/www/html/redcap/modules/mass_delete_v1.1.3/MassDelete.php(27): Stanford\MassDelete\MassDelete->render_page() 
 #2 /var/www/html/redcap/modules/mass_delete_v1.1.3/page_mass_delete.php(5): Stanford\MassDelete\MassDelete->init_page() 
 #3 /var/www/html/redcap/redcap_v12.0.25/ExternalModules/index.php(126): require('...') #4 {main} thrown in /var/www/html/redcap/modules/mass_delete_v1.1.3/MassDelete.php on line 229 

[27-Sep-2022 09:51:26 UTC] PHP Fatal error: Uncaught TypeError: Cannot access offset of type string on string in /var/www/html/redcap/modules/mass_delete_v1.1.3/MassDelete.php:229 

Stack trace: #0 /var/www/html/redcap/modules/mass_delete_v1.1.3/MassDelete.php(54): Stanford\MassDelete\MassDelete->handleDelete() #1 /var/www/html/redcap/modules/mass_delete_v1.1.3/MassDelete.php(27): Stanford\MassDelete\MassDelete->render_page() #2 /var/www/html/redcap/modules/mass_delete_v1.1.3/page_mass_delete.php(5): Stanford\MassDelete\MassDelete->init_page() #3 /var/www/html/redcap/redcap_v12.0.25/ExternalModules/index.php(126): require('...') #4 {main} thrown in /var/www/html/redcap/modules/mass_delete_v1.1.3/MassDelete.php on line 229
```

**Comment by tertek** [comment]
I have made a quick fix and tested the code with PHP8.1.
Can you please check?

https://github.com/susom/mass-delete/pull/11

**Comment by tertek** [comment]
Fixed in #11
